### ISSUE TITLE
Error while deserializing a RedeliveryStamp: FlattenException gets null values

### ISSUE THREAD

**Issue by Jean85** [symfony/symfony#38966]
**Symfony version(s) affected**: encountered on 5.1.7

**Description**  
I'm using Symfony Messenger with Doctrine as a failure transport, and the Symfony Messenger serializer to obtain JSON payloads.

I get a malformed `RedeliveryStamp`  when deserializing for some unknown reason (I'm still debugging).
These are the results of my debugging:
 * I got the error on my staging environment, where I use a different transport, so it's not the transport's fault
 * I went down to the DB, and the stamps are correctly serialized
 * I went down in the `deserialize` call chain, down to `\Symfony\Component\Serializer\Encoder\JsonDecode::decode` and the `json_decode` result seems ok, with the `flattenException` with all the values popolated:

```
{
    "type": "https:\/\/tools.ietf.org\/html\/rfc2616#section-10",
    "title": "An error occurred",
    "status": 401,
    "detail": "Error message from my exception - HTTP 401: <?xml version=\"1.0\"?>\n<response><error><message>Unauthorized<\/message><\/error><\/response>\n",
    "class": "Some\\Valid\\Class",
    "trace": ['...']
}
```
 * The error seems to lie in the denormalization step, where the stamp gets rehydrated

I'm currently at a loss, I cannot debug this further, the serializer is really too hard to follow... Any help?

**How to reproduce**  
TBD

**Comment by xabbuh** [comment]
What is the error that you get?

**Comment by xabbuh** [comment]
Possibly the same as #38792?

**Comment by Jean85** [comment]
Yes it seems the same! Closing as duplicate.
